src/main/java/com/google/sps/Interactions.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.logging.Logger;
+
+public class Interactions {
+
+  private static final Logger LOGGER = Logger.getLogger(Interactions.class.getName());
+
+  // contributions to user's interest metrics for each action
+  public static final int VIEW_SCORE = 1;
```

Comment:
How are these scores used?

Reply:
I haven't really gotten to this yet, but I'm hoping to use them as a way to keep track of a user's interest in each topic (a larger score would suggest a greater interest) and these scores would be incremented when users take different actions.